src/helpers/highlightActiveRow.js

```diff
@@ -0,0 +1,20 @@
+let activeRow = null;
+
+export const highlightActiveRow = (e) => {
+    if(activeRow) {
+        activeRow.style.backgroundColor = 'inherit'; // NOTE: Setting the style directly, because if you add a class, the internal styles of the library will override its properties.
```

Comment:
Setting the style directly, because if you add a class, the internal styles of the library will override its properties. Is this okay?